assets/src/scss/layout/_navbar.scss

```diff
@@ -69,12 +69,27 @@ $navbar-default-height: 60px;
       }
     }
 
-    height: 44px;
-    line-height: 40px;
+    height: $sp-5x;
+    line-height: $sp-5;
+
+    &.dropdown {
```

Comment:
This chevron is currently also showing in the mobile side menu, I don't think it should?

<img width="467" alt="Screenshot 2022-02-25 at 15 01 37" src="https://user-images.githubusercontent.com/6949075/155729331-871213be-acb4-48b4-b16e-8239dda61291.png">